docs/use/vscode.md

```diff
@@ -513,6 +513,53 @@ If multiple terminals are shared, or your focus is in a different tab, you can b
 
 To end your terminal session, simply type exit, close the terminal window, or click the "Unshare terminal" icon in the Live Share Explorer view or custom tab and everyone will be disconnected.
 
+## Enable accessibility features
+
+By default, Live Share keeps you aware of what other participants in the session are doing with visual indicators - a cursor to match each participant's location, a shared viewport in follow mode, synchronized edits, and so on. Live Share also has a setting to surface this information audibly, `liveshare.accessibility.accessibilityFeaturesEnabled`. With accessibility features enabled, Live Share plays various sounds and messages to keep you aware of what other participants are doing in a non-disruptive way.
+
```

Comment:
I would like to add a [!Note] here which calls out the following:
In implementing these accessibility features for the blind and visually impaired community, Live Share leveraged research from the Microsoft Research team, on mixed-ability collaboration. The recommendations from this research were then specifically implemented for the use of pair-programming with Live Share.

Reply:
Is documentation the appropriate place to put that reference? I definitely feel like it should be included *somewhere*, but I don't know if that should be *here*.

Reply:
I think this can be the place to permanently call this out. We often have used documentation to call out new features and link to articles etc. Once more widely known we can remove this if needed.

Reply:
What's the point of calling this out? Does it add anything for the user that would allow them to get more out of the feature? Perhaps if we can link to more resources. But at the moment we can't do that.